### PR TITLE
Add a blog post for the 1.6 release

### PR DESCRIPTION
**Pull request by annajung**
Signed-off-by: Anna Jung (VMware) <antheaj@vmware.com>

- Add a blog post for the 1.6 release

cc @kubeflow/release-team @jbottum

### CODE REVIEW

**Comment by jbottum** [comment]
@johnugeorge do the Training Operator Python SDK and Clientset support tensorflow too ?

**Comment by jbottum** [comment]
@kimwnasptd could you please review the Notebook and Manifest section ?

**Comment by annajung** [comment]
deploy preview: https://deploy-preview-124--infallible-murdock-d05108.netlify.app/kubeflow-1.6-release/

**Comment by jbottum** [comment]
@pvaneck would you please review the KServe info in this post?

**Comment by jbottum** [comment]
@james-jwu @zijianjoy would you guys please review the KFP content and make sure the links are correct ?

**Comment by jbottum** [comment]
> @johnugeorge do the Training Operator Python SDK and Clientset support tensorflow too ?

per our conversation on 8/25, tensorflow is supported in both and the other frameworks are the enhancement.

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (shivaylamba) [inline _posts/2022-08-24-kubeflow-1.6-release.md:128]: Adding a full stop at end of the line

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (shivaylamba) [inline _posts/2022-08-24-kubeflow-1.6-release.md:32]: small case for community

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (pvaneck) [inline _posts/2022-08-24-kubeflow-1.6-release.md:117]: Suggest having this list item be:

`Multi-namespace support for the ModelMesh alternative backend`

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: Added . at the end

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md:32]: done

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: Updated

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (jbottum) [inline _posts/2022-08-24-kubeflow-1.6-release.md:53]: The pipelines WG says that existing lines 52 and line 53 should be deleted and replace with this line...

[Streamlined Component Authoring] (https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_438)

Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: Done

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (johnugeorge) [inline _posts/2022-08-24-kubeflow-1.6-release.md:73]: URL ?

Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: In the original google docs, it looks like the word "URL" was added to propose adding a link to the paper here. I removed the word "URL" and if you want to provide a paper that you want to link, I can link it here

**Comment by jbottum** [comment]
Some proposed changes.  @annajung I am still having trouble making edits inline, not sure why but maybe you can review these proposed changes.

Line 2 (to use alpha instead of experimental)
Kubeflow v1.6 delivers support for Kubernetes v1.22 and introduces an alpha release of Kubeflow Pipeline v2 functionality

Line 13-14 (added words to make more readable)
The Kubeflow Community is excited to announce the availability of the Kubeflow v1.6 software release, which includes support for Kubernetes v1.22 and introduces an alpha release of Kubeflow Pipeline v2 (KFP v2) functionality.

Line 19-21 (note - this includes deleting some current content and replacing with below)
Additionally, 1.6 cleans up a few CVEs in the central dashboard and enables PodDefaults webhook to pick-up new certificate updates. For the Kubernetes upgrade, the community developed and tested each Kubeflow component manifest for Kubernetes v1.22. As v1.22 introduced some breaking changes, the upgrade was a team effort, and this Kubeflow release management process will be useful in the community's future updates of Kubernetes and other software dependencies.

Line 35 (just adding v to 1.6)
To round back to the current software delivery, the following takes a deeper look at the v1.6 highlights in more detail.

Line 41 (change name of slide deck to match the name of the slide deck)
change from "Understand KFP" to "KFP v2 Introduction"

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (jbottum) [inline _posts/2022-08-24-kubeflow-1.6-release.md:41]: changed name of slide deck to match the actual name of the presentation (KFP v2 Introduction)

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (jbottum) [inline _posts/2022-08-24-kubeflow-1.6-release.md:35]: add v to v1.6

**Review by jbottum — CHANGES_REQUESTED** [review]
proposed changes in several places

**Comment by jbottum** [comment]
Future To Do: We should consider adding a link in the blog post to the Kubeflow 1.6 video presentation.   We do not have that link, which will be on Youtube.    Once that link is up, we should consider adding it to the blog post, perhaps at the end of the summary paragraph near the top to the post i.e. ~ The Kubeflow Working Group leads and Release Team have recorded a review of Kubeflow v1.6's features, which you can view here (link TBD).

**Comment by jbottum** [comment]
@johnugeorge @zijianjoy @kimwnasptd @annajung I am trying to finish this up for posting, target Thursday.   Could you please provide your /lgtm ?   thanks.

**Comment by annajung** [comment]
lgtm after Josh's changes

**Review by shivaylamba — APPROVED** [review]

**Review by charlesa101 — APPROVED** [review]

**Review by terrytangyuan — APPROVED** [review]
LGTM. Thanks everyone from the release team!

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (chensun) [inline _posts/2022-08-24-kubeflow-1.6-release.md:56]: I think the slide "Understanding KFP v2" [1] is still a good overall reference for learning more about KFP v2.
In comparison, "KFP SDK V2 CLI design" is not essential to KFP v2, and can be skipped in this context. 

[1] https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gb4a3fac3a8_7_1911

Other than this, LGTM. Thanks!

Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: Replaced the KFP SDK v2 CLI design link with the KFP v2 introduction link

**Comment by chensun** [comment]
/lgtm

Thanks!

@james-jwu for approval.

File: _posts/2022-08-24-kubeflow-1.6-release.md
Comment (jbottum) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: @annajung lines 41 and lines 56 both refer to users to the KFP v2 Introduction slide deck.   We might delete line 56.

**Pull request merged**
